src/eval-frame/components/reps/rep-utils/get-data-table-summary.js

```diff
@@ -0,0 +1,44 @@
+import { range } from "lodash";
+import { getValueSummary } from "./get-value-summary";
+
+import { getObjAtPath } from "./get-child-summaries";
+
+export function getDataTableSummary(rootObjName, path, pageSize, page) {
```

Comment:
this file/function is about producing the summary data to show a tabular rep remotely